java21-openjdk/riscv64.patch

```diff
@@ -0,0 +1,52 @@
+--- PKGBUILD
++++ PKGBUILD
+@@ -17,22 +17,25 @@ pkgrel=1
+ arch=('x86_64')
+ url='https://openjdk.java.net/'
+ license=('LicenseRef-Java')
+-makedepends=('java-environment=21' 'cpio' 'unzip' 'zip' 'libelf' 'libcups' 'libx11'
++makedepends=('java-environment=22' 'cpio' 'unzip' 'zip' 'libelf' 'libcups' 'libx11'
+              'libxrender' 'libxtst' 'libxt' 'libxext' 'libxrandr' 'alsa-lib' 'pandoc'
+              'graphviz' 'freetype2' 'libjpeg-turbo' 'giflib' 'libpng' 'lcms2'
+              'libnet' 'bash' 'harfbuzz' 'gcc-libs' 'glibc')
+ source=(https://github.com/openjdk/jdk${_majorver}u/archive/${_git_tag}.tar.gz
+         freedesktop-java.desktop
+         freedesktop-jconsole.desktop
+-        freedesktop-jshell.desktop)
++        freedesktop-jshell.desktop
++        bootstrap-using-jdk22.patch)
```

Comment:
Is this patch only used in bootstrap? If true, would splitting a `java21-openjdk-bootstrap` package be better?

Reply:
The bootstrap process is a) build Java 21 using 22 b) build 21 again using 21.

Reply:
I think it's fine here since openjdk 22 is likely to be gone fairly soon :(